test/Microsoft.AspNetCore.Mvc.Abstractions.Test/project.json

```diff
@@ -13,6 +13,7 @@
     "netcoreapp1.0": {
       "imports": [
         "dnxcore50",
+         ,
```

Comment:
Looking back at my terminal history it looks like after I did a find and replace I tried to do a build to check that it all worked, but built the wrong repo without noticing. Cleaning all these up now.